retrieve-logs/README.md

```diff
@@ -0,0 +1,77 @@
+# How to replay missing Matomo requests to Matomo
+
+These steps describe how to get missing Matomo requests from AWS CloudWatch.
+Once obtained, the logs can be used to replay the requests to Matomo.
+
+You will need to authenticate with AWS via the command line to use this script.
+Namely, you will need to set the following environment variables to appropriate values:
+
+- `AWS_ACCESS_KEY_ID`
+- `AWS_SECRET_ACCESS_KEY`
+- `AWS_SESSION_TOKEN` - this is obtained from the AWS Security Token Service
+- `AWS_DEFAULT_REGION`
+
+## Steps
+
+1. Authenticate for the AWS CLI
+2. Run the following command to build a docker image.
+    ```
+    docker build -t matomo-retrieve-logs .
+    ```
+3. Run the following command to get the file containing missing Matomo requests. You need to specify START_DATE and NUM_OF_DAYS before you run the command.
+    ```
+    docker run -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY -e AWS_SESSION_TOKEN=$AWS_SESSION_TOKEN -e AWS_DEFAULT_REGION=$AWS_DEFAULT_REGION -e START_DATE=2019-01-01T00:00:00+00:00 -e NUM_OF_DAYS=1 -v $(pwd):/app -it --rm --name my-running-app matomo-retrieve-logs
```

Comment:
The `--name my-running-app` bit not required ... seems a bit arbitrary.

Reply:
😕 I thought I'd addressed this in #3 . If not, I'll remove it. 👍

Reply:
Removed in https://github.com/alphagov/verify-matomo-utils/pull/3/commits/8182d6c3af524058544e202eef54c45782064164